src/test/java/mate/academy/dao.impl/UserDaoImplTest.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.dao.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoImplTest extends AbstractTest {
+    private static final String BOB_EMAIL = "bob@gmail.ua";
+    private static final String ALICE_EMAIL = "alice@gmail.ua";
+    private static UserDaoImpl userDao;
+    private Set<Role> roles;

```

Comment:
you use it one test method, make it local. Look through other classes and check if you can to do the same for other fields